crates/oxc_language_server/src/capabilities.rs

```diff
@@ -92,12 +112,20 @@ mod test {
                 }),
                 ..TextDocumentClientCapabilities::default()
             }),
+            workspace: Some(WorkspaceClientCapabilities {
+                workspace_edit: Some(WorkspaceEditClientCapabilities {
+                    document_changes: Some(true),
+                    ..WorkspaceEditClientCapabilities::default()
+                }),
+                ..WorkspaceClientCapabilities::default()
+            }),
             ..ClientCapabilities::default()
         };
 
         let capabilities = Capabilities::from(client_capabilities);
 
         assert!(capabilities.code_action_provider);
+        assert!(capabilities.workspace_edit);
```

Comment:
Could you please create own tests for `workspace_execute_command`.
Feel free to create your own tests for nvim too.

Please use right config from the program.

VsCode:
```
                    execute_command: Some(
                        DynamicRegistrationClientCapabilities {
                            dynamic_registration: Some(
                                true,
                            ),
                        },
                    ),
```

IntelliJ:
```
    executeCommand = ExecuteCommandCapabilities [
      dynamicRegistration = false
    ]
```

Reply:
My bad, I didn't read the test name :sweat_smile: I thought it was a general editor capability test.